src/plugins/data/public/ui/filter_bar/filter_options.tsx

```diff
@@ -28,7 +28,13 @@
  * under the License.
  */
 
-import { EuiButtonIcon, EuiContextMenu, EuiPopover, EuiPopoverTitle } from '@elastic/eui';
+import {
+  EuiButtonIcon,
```

Comment:
This was EuiSmallButtonIcon in the previous diff - why change it in the backport?